src/common/components/modal/modal.module.scss

```diff
@@ -5,6 +5,7 @@
 @import 'variables';
 
 $modalPadding: 2rem;
+$lgModalPadding: 3rem;
```

Comment:
this is quite unnecessary, just use $largePadding

Reply:
Ah, yes!